ROMFS/px4fmu_common/init.d/4900_crazyflie

```diff
@@ -0,0 +1,38 @@
+#!nsh
+#
+# @name Crazyflie config
```

Comment:
This appears in the QGC airframe list as "Crazyflie config". Better would be "Crazyflie 2".